packages/favorites/__test__/FavoriteHeart.test.js

```diff
@@ -1,6 +1,11 @@
 import React from 'react';
 import 'react-testing-library/cleanup-after-each';
```

Comment:
We already have the `cleanup` running after each run. You shouldn't have to import the `cleanup` function as well. Since you also have to clear the mocks just remove this line as well or else its running the cleanup function twice.

Reply:
918addb1072df98218d43a018947d24267f30254

good catch - forgot to remove when i was trying to determine why i had to move that test.